recipes/rnaredprint/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version="0.3pre" %}
+{% set sha256="97e175d521eabdc1124ec93cc78b5f8e39c8bae34e2cb4cd403c0005ebf86233" %}
+{% set vrnaversion="2.4" %}
+
+package:
+  name: rnaredprint
+  version: {{ version }}
+
+about:
+  home: https://github.com/yannponty/RNARedPrint
+  license: GPL
+  license_file: LICENSE
+  summary: Tree-decomposition based dynamic programming algorithm for multiple target RNA design
+
+build:
+  number: 0
+
+source:
+  url: "https://github.com/yannponty/RNARedPrint/archive/v{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python <3
```

Comment:
Wouldn't `python <3` be required for host and/or run modes as well?

Reply:
good catch, I am not sure that we even still have the python build-dependency. I'll check it.
To run, we need python3